eksconfig/add-on-node-groups.go

```diff
@@ -68,6 +76,8 @@ type ASG struct {
 	// TODO: handle conflicting flag '--cloud-provider aws'
 	// ref. https://github.com/kubernetes/kubernetes/issues/64659
 	KubeletExtraArgs string `json:"kubelet-extra-args"`
+	// clusterautoscaler per nodegroup
+	ClusterAutoScaler *NGClusterAutoScaler `json:"cluster_auto_scaler,omitempty"`
```

Comment:
To be consistent with other JSON tags, let's just name it `json:"cluster-autoscaler,omitempty"`